packages/replay/src/replay.ts

```diff
@@ -148,7 +147,6 @@ export class ReplayContainer implements ReplayContainerInterface {
     this.timeouts = {
       sessionIdlePause: SESSION_IDLE_PAUSE_DURATION,
       sessionIdleExpire: SESSION_IDLE_EXPIRE_DURATION,
-      maxSessionLife: MAX_SESSION_LIFE,
```

Comment:
Why not keep the max duration part of timeouts?

Reply:
🤷 just aligning this - the only reason these `timeouts` exist was to allow overwriting them in tests, which can now be done "correctly"/publicly via `maxReplayDuration`?